pkgs/by-name/as/astrometry-net/package.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, python3, cairo, netpbm, libpng, libjpeg, zlib, bzip2, swig, cfitsio, pkg-config, gsl }:
+
+stdenv.mkDerivation (FinalAttr: {
+  pname = "astrometry.net";
+  version = "0.94";
+  version_date = "2023-05-01";
+
+  src = fetchFromGitHub {
+    owner = "dstndstn";
+    repo = FinalAttr.pname;
+    rev = FinalAttr.version;
+    hash = "sha256-/g9cqgcoxYQAmSdNrdAN/ZkQBu2xynOFYzfZnhKgKF8=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ python3 cairo netpbm libpng libjpeg zlib bzip2 swig cfitsio gsl ];
+  propagatedBuildInputs = with python3.pkgs; [ numpy ];
+
+  preConfigure = ''
+    patchShebangs .
+  '';
+
+  makeFlags = [ "INSTALL_DIR=$(out)" "AN_GIT_REVISION=${FinalAttr.version}" "AN_GIT_DATE=${FinalAttr.version_date}" ];
```

```suggestion
  makeFlags = [
    "INSTALL_DIR=$(out)"
    "AN_GIT_REVISION=${FinalAttr.version}"
    "AN_GIT_DATE=1970-01-01"
  ];
```
mocking the date is making updates a lot easier